### PR TITLE
wsdd2: update to 2023-12-21

### PR DESCRIPTION
**Pull request by Lucky1openwrt**
@neheb
Compile tested: (ipq806x,Linksys EA7500,V23.05.2)
Compile tested: (ramips,Linksys EA8100,V23.05.2)
Compile tested: (ramips,D-Link DIR-1960-A1,V23.05.2)
Run tested: (ipq806x,Linksys EA7500,V23.05.2)
Run tested: (ramips,Linksys EA8100,V23.05.2)
Run tested: (ramips,D-Link DIR-1960-A1,V23.05.2)

Description:
finally make this thing work as it hasn't since V19

after an interface changes "DHCP renew or other"
successfully reload with new interface alterations

https://github.com/Netgear/wsdd2/pull/46
https://github.com/Netgear/wsdd2/issues/36

### CODE REVIEW

**Comment by Lucky1openwrt** [comment]
same or slimier to 
https://github.com/openwrt/packages/pull/22550/files

**Comment by 1715173329** [comment]
>  WARNING: PKG_MIRROR_HASH does not match wsdd2-2023-12-21-b676d8ac.tar.xz hash ac817f63605508779ebebf612fcb7d594ff87d3050b788f240d1fea9fdbaa3a0

 CI is failing. We expect lower case hash value.

**Comment by Lucky1openwrt** [comment]
Thanks I have changed it
but now know my build system is not checking it
so I can't be sure it passes

File: net/wsdd2/Makefile [hunk 1]
```diff
@@ -5,9 +5,9 @@ PKG_RELEASE:=3
```
Comment (BKPepe) [inline net/wsdd2/Makefile:5]: PKG_RELEASE needs to be resetted.

Comment (Lucky1openwrt) [reply net/wsdd2/Makefile:5]: I'm not sure what this means 
if it's about the build system 
I erased everything 
update and downloaded feeds
change date & version only NOT Hash
then compiled 
it downloads new file into dl & compiles ok with out errors at all 
I could not get it to fail :(

Comment (BKPepe) [reply net/wsdd2/Makefile:5]: I said that you should change only the value from 3 into 1. Dont reinvent a wheel, dude.

Comment (Lucky1openwrt) [reply net/wsdd2/Makefile:5]: ok thanks will do

**Comment by Lucky1openwrt** [comment]
thanks :)

**Pull request merged**